packages/plugins/Filter/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## 1.1.2
```

Comment:
Shouldn't this be in `unpublished` at first?

Reply:
Yep, fixed that.